.gitignore

```diff
@@ -1,5 +1,6 @@
 # Directories that are generated
 /tags
+cmake-build*
```

Comment:
We needed this to avoid having every build file show up when we run `git status`. This is already done in libmongoc